pkg/pillar/pubsub/pubsub.go

```diff
@@ -53,7 +60,7 @@ type keyMap struct {
 }
 
 // PubSub is a system for publishing and subscribing to messages
-// it manages the creation of Publication and Subscription, which handle the actual
+// it manages the creation of NewPublication and NewSubscription, which handle the actual
```

Comment:
Same as before, revert comment. We are creating a `Publication` or a `Subscription`, the things; `NewPublication` and `NewSubscription` are just func calls that create, respectively, `Publication` and `Subscription`.